<launch>
  <!--ros_args.launch  to plot_gui-->
  <arg name="plot_gui" default="true" doc="rqt_plot~to plot the values of the ~pose_error"/>

  <!--start the turtlesim and turtle_rect nodes-->
  <node pkg="turtlesim" name="turtle" type="turtlesim_node"/>
  <node pkg="tsim" name="rectmove" type="turtle_way" output="screen"/>

  <!--If the plot_gui is true, start the gui to plot the pose_error-->
  <group if="$(arg plot_gui)">
    <node name="error_plot" pkg="rqt_plot" type="rqt_plot" args="/pose_error" required="true"/>
  </group>

</launch>
